docs/writing-tests.md

```diff
@@ -41,7 +41,19 @@ In contrast, the tests in this repo provide coverage at a higher level. We shoul
 
 ## Adding new tests
 
-We use feature files to describe single applications (e.g. [`whitehall`](https://github.com/alphagov/whitehall), [`frontend`](https://github.com/alphagov/frontend)), as well as [cross-application behaviour](https://github.com/alphagov/smokey/blob/main/features/gov_uk.feature).
+Scenarios should be organised by the "component" they are testing:
+
+- **`<app>.feature`** should contain tests that target a feature of that app e.g.
```

Comment:
Following 7ddeb78cab1fe160de5455e1ea03f93cd3e3ffaa where you moved the app-focussed features into an `app/` subdirectory, it'd be good to update this guidance to match.

And then as you said in https://github.com/alphagov/smokey/pull/976#issuecomment-1160318013, it sounds like you're suggesting the `<layer>.feature` files remain as top level features.